src/pages/case-studies/overview.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Overview
+description:
+  We’re currently developing case studies to explore the adoption and
+  implementation of Carbon in products and digital experiences.
+---
+
+<PageDescription>
+
+We’re currently developing case studies to explore the adoption and
```

Comment:
Can we just say "We're developing case studies..."? The "currently" seems redundant.

Reply:
Fixed!